extensions/wikia/WallNotifications/mainetannce/cleanupWallNotifications.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+/**
+ * Script that removes dataware.wall_notification entries that are old than X days
+  *
+ * @author macbre
+ * @file
+ * @ingroup Maintenance
+ */
+
+require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
+
+/**
+ * Maintenance script class
+ */
+class CleanUpWallNotifications extends Maintenance {
+
+	const WALL_NOTIFICATION_TABLE = 'wall_notification';
+	const DELETE_BATCH = 200000; // takes 1-2 seconds
+
+	const DAYS_DEFAULT = 90; // remove notifications older than X days, can be customized via --days option
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)